tests/unittest_inference.py

```diff
@@ -4543,7 +4543,7 @@ def test_lambda(self) -> None:
         )
         inferred = next(node.infer())
         self.assertIsInstance(inferred, nodes.Const)
-        self.assertIs(inferred.value, False)
+        self.assertFalse(inferred.value)
```

Comment:
This one feels wrong because we already check with a specialized function and ``inferred.value`` could be falsey.

Reply:
Yeah, this is incorrect, see: https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertFalse.

Makes me wonder whether we should really use this.

I have looked into https://github.com/dannysepler/pytestify and similar tools but thought it wasn't worth the churn to apply them.

Reply:
Migrating to pytest's assertions actually make more sense.